Mage.Server/src/test/java/mage/server/util/ConfigWrapperTest.java

```diff
@@ -0,0 +1,294 @@
+package mage.server.util;
+
+import mage.server.util.config.*;
+import mage.utils.FluentBuilder;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.RandomUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.TestFactory;
+
+import java.math.BigInteger;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ConfigWrapperTest {
+
+    static class ConfigBuilder extends FluentBuilder<Config, ConfigBuilder> {
```

Comment:
The build is used to make the tests as terse as possible. All access to all components of the configuration is verified later.